packages/get-platform/src/getPlatform.ts

```diff
@@ -18,16 +17,33 @@ const supportedLibSSLVersions = ['1.0.x', '1.1.x', '3.0.x'] as const
 
 // https://www.geeksforgeeks.org/node-js-process-arch-property/
 export type Arch = 'x32' | 'x64' | 'arm' | 'arm64' | 's390' | 's390x' | 'mipsel' | 'ia32' | 'mips' | 'ppc' | 'ppc64'
+export type DistroInfo = {
+  /**
+   * The original distro is the Linux distro name detected via its release file.
+   * E.g., on Arch Linux, the original distro is `arch`. On Linux Alpine, the original distro is `alpine`.
+   */
+  originalDistro?: string
+
+  /**
+   * The family distro is the Linux distro name that is used to determine Linux flavors based on the same base distro, and likely using the same package manager.
+   * E.g., both Ubuntu and Debian belong to the `debian` family of distros, and thus rely on the same package manager (`apt`).
+   */
+  familyDistro?: string
+
+  /**
+   * The target distro is the Linux distro associated with the Prisma Engines.
+   * E.g., on Arch Linux, Debian, and Ubuntu, the target distro is `debian`. On Linux Alpine, the target distro is `musl`.
+   */
+  targetDistro?: 'rhel' | 'debian' | 'musl' | 'arm' | 'nixos' | 'freebsd11' | 'freebsd12' | 'freebsd13'
+}
 export type GetOSResult = {
   platform: NodeJS.Platform
   arch: Arch
-  distro?: 'rhel' | 'debian' | 'musl' | 'arm' | 'nixos' | 'freebsd11' | 'freebsd12' | 'freebsd13'
-
   /**
    * Starting from version 3.0, OpenSSL is basically adopting semver, and will be API and ABI compatible within a major version.
    */
   libssl?: typeof supportedLibSSLVersions[number]
-}
+} & Partial<DistroInfo>
```

Comment:
DistroInfo is already partial, so I think you can remove this

Reply:
Also, I could suggest making a `distro: DistroInfo` entry, but not a hard requirement. The you would do `distro.family` instead of `familyDistro`.

Reply:
That'd be a nice to have, but as you say, no need to do it until it's needed. Let's merge this PR first

Reply:
`Partial` isn't needed though

Reply:
Right, removed.